src/back/msl/writer.rs

```diff
@@ -1767,6 +1816,15 @@ impl<W: Write> Writer<W> {
                 tyvar.try_fmt(&mut self.out)?;
                 writeln!(self.out, "{}", separator)?;
             }
+
+            if options.sizes_buffer_binding.is_some() {
+                writeln!(
+                    self.out,
+                    "{}constant mslBufferSizes& x_buffer_sizes",
```

Comment:
why is the `x_` prefix?

Reply:
just start it with an underscore then. We use "_eXxx"  for expressions, for example